devenv/loadtest/modules/client.js

```diff
@@ -1,5 +1,78 @@
 import http from "k6/http";
 
+export const DatasourcesEndpoint = class DatasourcesEndpoint {
+  constructor(httpClient) {
+    this.httpClient = httpClient;
+  }
+
+  getAll() {
+    return this.httpClient.get('/datasources');
+  }
+
+  getById(id) {
+    return this.httpClient.get(`/datasources/${id}`);
+  }
+
+  getByName(name) {
+    return this.httpClient.get(`/datasources/name/${name}`);
+  }
+
+  create(payload) {
+    return this.httpClient.post(`/datasources`, JSON.stringify(payload));
+  }
+
+  update(id, payload) {
+    return this.httpClient.put(`/datasources/${id}`, JSON.stringify(payload));
+  }
+
+  delete(id) {
```

Comment:
I don't know if we'll really need this endpoint one day. I don't think we should delete things in case we're running our load tests against a dev instance with already some data in it.

Reply:
Removed.